crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -0,0 +1,568 @@
+let PI: f32 = 3.141592653589793;
+
+fn saturate(value: f32) -> f32 {
```

Comment:
It think we should prefix all these functions with a prefix like `bevy_pbr_functions` so that we don't pollute the global namespace.

Reply:
Given we have to use binding variables directly, I was thinking that they should perhaps also be prefixed. I agree, I think it's a good idea.

Reply:
I feel like for things like PI and saturate we could have a bevy_shader_utils.wgsl or something like that because those aren't specific to pbr and could be useful in plenty of other situations

Reply:
Sounds good. I'd be inclined to call it `math.wgsl` or `common.wgsl`.